src/main/java/org/minions/devfund/angela/Squarelotron.java

```diff
@@ -0,0 +1,128 @@
+package org.minions.devfund.angela;
+
+import java.util.Arrays;
+
+/**
+ * Manage squarematrix.
+ */
+public class Squarelotron {
+
+    private int[][] squarematrix;
+    private int size;
+
+    /**
+     * Constructor for {@link Squarelotron}.
+     *
+     * @param dimension the dimension to fill the squarematrix.
+     */
+    public Squarelotron(final int dimension) {
+        size = dimension;
+        squarematrix = new int[dimension][dimension];
+        int counter = 0;
+        for (int i = 0; i < dimension; i++) {
+            for (int j = 0; j < dimension; j++) {
+                counter++;
+                getSquarematrix()[i][j] = counter;
```

Comment:
if you have the attribute of class you can use it directly
Don't use the getter
squarematrix[i][j] = counter;